sale_order_invoice_qty_number/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, fields, api
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    invoice_ids = fields.Many2many("account.move")
```

Comment:
Even if it's used, an already existing field shouldn't be redefined, only if it has changed, and that's not the case. So this line should be removed